### ISSUE TITLE
eSpeak-ng detects a sentence only if the next one starts with a capital letter

### ISSUE THREAD

**Issue by vortex1024** [espeak-ng/espeak-ng#872]
The title is pretty confusing, so I will give an example:
"This is the list. it is biggern than I thought"
Because the second sentence doesn't start with a capital letter, the dot doesn't cause a pause between sentences. Gramatically speaking, it is correct, but in informal chats and so, people frequently write like that, and reading what they say is difficult.
I am thinking the behaviour might be intentional, for abbreviations and so, since only the dot sign is affected. It works fine for ! and ?. . I suggest it should be at least made optional, if not disabled.
Thoughts?
Thanks.

**Comment by jaacoppi** [comment]
I agree. I think all three (. ! ?) should be treated equally for ending sentences. Dots should end the sentence if an abbreviation is not detected. Can anyone think of use cases that benefit from the current behaviour?

P.S.
Handling dots / periods is one of the most confusing parts of espeak-ng. Just try espeak-ng "this.useCase(test).example() == null" - somehow it recognises the dot and speaks it aloud. That is very useful for programming.

**Comment by sukiletxe** [comment]
tbh I use this to detect typos (you guessed it, missing capital letters), but yes it makes text complicated to read.

**Comment by jaacoppi** [comment]
https://github.com/espeak-ng/espeak-ng/blob/4838b356022c3b10dbc05acca12e015e4f64bf0d/src/libespeak-ng/readclause.c#L1031-L1034

Removing those lines would treat . followed by a lowercase letter as a sentence terminator. However, that would break current tests.

Some language, Finnish for example, use the form abbrev + dot for shortening words. For example "jne." is pronounced "ja niin edelleen", "pvm." is short for "päivämäärä", "huom." is short for "huomio" and so on. Removing the lines above would break that behaviour. Maybe the next word could be checked against dictionary flags $abbrev, $dot and $hasdot could work.

**Comment by jaacoppi** [comment]
> They advised their use case for this was that they do not use capitals in Georgia (I believe the European country rather than the US state).

In that case we really should have an option to choose whether to pause on capitals only or also with lowercase.

The change requires:
1. read the option in voices.c / LoadVoice()
2. check for the option in readclause.c / ReadClause()
3. add the option to Georgian voice file espeak-ng-data/lng/ccs/ka
4. document the option in docs/voices.md
5. add a test
6. collect feedback from users to see if it really works as intended

I'm busy at the moment but hopefully I'll be able to implement this in April.

**Comment by jaacoppi** [comment]
I just pushed https://github.com/espeak-ng/espeak-ng/commit/c4740b30533ee0010b6adf23b25483f14bcbbc47 and https://github.com/espeak-ng/espeak-ng/commit/cad1c8e87fcccf677a445202e340f61980450a84. They should be enough to close this issue. 

@vortex1024: add "lowercaseSentence" to your language files in espeak-ng-data/lang/* and see if this is exactly what you wanted or if there should be something else.

@Qchristensen: The option is now set on by default for Georgian, but not any other language. Please gather feedback from your users or point them here directly.
